pyanaconda/installation.py

```diff
@@ -319,6 +320,16 @@ def run_pre_install():
         late_storage.append_dbus_tasks(STORAGE, [conf_task])
         installation_queue.append(late_storage)
 
+    # Add installation tasks for the Subscription DBus module
+    # - to see if this is appropriate for the current environment we check if
+    #   the Subscription module is running via a helper function
+    if check_subscription_module_available():
+        subscription_config = TaskQueue("Subscription configuration",
+                                        N_("Configuring Red Hat subscription"))
+        subscription_proxy = SUBSCRIPTION.get_proxy()
+        subscription_dbus_tasks = subscription_proxy.InstallWithTasks()
```

Comment:
It there a reason why this task should be run here? Couldn't we move it at the beginning of `_prepare_configuration`?

Reply:
I simply put it to the same location where it was in RHEL 8.2. It needs storage to be configured and mounted and packages to be installed (in case the Insights task runs). Given that tasks added in `_prepare_configuration` run even later than where it is now it should be fine to move it there & I'll do that. :)